components/GrantApplication.vue

```diff
@@ -6,170 +6,175 @@
+              :class="{
+                'invisible-input': currentApplication.orgBudget
+              }"
+              :disabled="!!currentApplication.orgBudget"
+              @change="onFileUpload($event, 'orgBudget')"
+              type="file"
+              name="orgBudget"
+            />
+          </label>
+
+          <label for="irs-letter">
+            <div>IRS Letter of Determination</div>
+            <div v-if="currentApplication.irsLetter" class="uploaded">
+              <i class="fa fa-check-square"></i>Uploaded!
+            </div>
+            <input
+              id="irs-letter"
+              :class="{
+                'invisible-input': currentApplication.irsLetter
+              }"
+              :disabled="!!currentApplication.irsLetter"
+              @change="onFileUpload($event, 'irsLetter')"
+              type="file"
+              name="irsLetter"
+            />
+          </label>
+        </fieldset>
         <div>
-          <p v-if="unfinished"><strong>The form is not finished</strong></p>
-          <p>Please fill in every field first before submitting</p>
-        </div>
-        <div v-if="needFile">
-          <p><strong>You have to upload at least one file</strong></p>
-          <p>Please upload at least one file</p>
-        </div>
-        <div v-if="uploading">
-          <p>
-            <i class="fa fa-spinner fa-pulse fa-2x"></i>Uploading...please wait
-          </p>
+          <div>
+            <p v-if="unfinished"><strong>The form is not finished</strong></p>
+            <p>Please fill in every field first before submitting</p>
+          </div>
+          <div v-if="needFile">
+            <p><strong>You have to upload at least one file</strong></p>
+            <p>Please upload at least one file</p>
+          </div>
+          <div v-if="uploading">
+            <p>
+              <i class="fa fa-spinner fa-pulse fa-2x"></i>Uploading...please
+              wait
+            </p>
+          </div>
+          <div v-if="completed">
+            <p><strong>Application completed!</strong></p>
+            <p>
+              Thank you! You have finished uploading all required files. Our
+              administrator will be in contact with you after reviewing the
+              files. If you have any questions or concerns please e-mail
+              info@vatterottfoundation.org.
+            </p>
+          </div>
         </div>
-        <div v-if="completed">
-          <p><strong>Application completed!</strong></p>
-          <p>
-            Thank you! You have finished uploading all required files. Our
-            administrator will be in contact with you after reviewing the files.
-            If you have any questions or concerns please e-mail
-            info@vatterottfoundation.org.
-          </p>
+        <div class="form-actions">
+          <button
+            type="submit"
+            class="primary"
+            ng-disabled="grantApp.messages.completed"
```

Comment:
Accidental typo, should be `:disabled` instead of the old AngularJS binding.

Reply:
Done